spk/sickbeard-custom/src/conf/resource

```diff
@@ -0,0 +1,5 @@
+{
+	"port-config":{
+    	"protocol-file": "etc/sickbeard-custom.sc"
```

Comment:
That was to be expected, I guess...
Does DSM6 still support the old servicetool command to add the service configure file? I'm thinking maybe drop this part, and see if we can integrate placing the file into the etc dir (or whatever) via the framework, instead of having to add two additional `_extra_install` steps for nearly every package. Best to keep the approach for something like that synced imo.